cmd/aries-agent-rest/startcmd/start.go

```diff
@@ -55,7 +54,7 @@ const (
 	databaseTypeEnvKey        = "ARIESD_DATABASE_TYPE"
 	databaseTypeFlagShorthand = "q"
 	databaseTypeFlagUsage     = "The type of database to use for everything except key storage. " +
-		"Supported options: mem, leveldb. " +
+		"Supported options: mem. " +
```

Comment:
Supporting only `mem` isn't good. We should import `-ext` and keep levelDB as default. There should be no problem importing `-ext` from _this_ module.

Reply:
While I agree, a LevelDB implementation of the new storage interface was moved out of this milestone, so I don't have one to use.

Reply:
@troyronda @fqutishat ^^

Reply:
@DRK3 Need a TODO to restore.

Reply:
@troyronda https://github.com/hyperledger/aries-framework-go/issues/2532